src/components/DatePickerInput/DatePickerInput.js

```diff
@@ -14,7 +14,6 @@ const DatePickerInput = ({ playlistDate, setPlaylistDate }) => {
       format="MM/DD/YYYY"
       margin="normal"
       id="date-picker-inline"
-      inputValue={playlistDate}
```

Comment:
Why did this get deleted?

Reply:
We need an inputValue and value for the datepicker, as far as I'm aware

Reply:
After testing with and without this prop, it worked either way. Happy to add back if I'm missing something. The initial value and input both feed from the value prop.

Reply:
Let me pull this down and double check

Reply:
Looks like you're right. Not sure what was making that behavior different initially. Nice job!